internal/sessiontest/revocation_test.go

```diff
@@ -196,9 +199,7 @@ func TestRevocationAll(t *testing.T) {
 		time.Sleep(time.Second)
 
 		// run scheduled update of accumulator, triggering a POST to our IRMA server
-		revServer.conf.IrmaConfiguration.Scheduler.RunAll()
-		// give request time to be processed
-		time.Sleep(100 * time.Millisecond)
```

Comment:
This line should be reinstated as well; its purpose was not to wait for the scheduler to finish its jobs, but instead to give time to finish running to a HTTP handler which is invoked by a job.